sycl/test/host-interop-task/interop-task-dependency.cpp

```diff
@@ -0,0 +1,203 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %threads_lib
+// RUN: %CPU_RUN_PLACEHOLDER SYCL_PI_TRACE=-1 %t.out 2>&1 %CPU_CHECK_PLACEHOLDER
+// RUN: %GPU_RUN_PLACEHOLDER SYCL_PI_TRACE=-1 %t.out 2>&1 %GPU_CHECK_PLACEHOLDER
+// RUN: %ACC_RUN_PLACEHOLDER SYCL_PI_TRACE=-1 %t.out 2>&1 %ACC_CHECK_PLACEHOLDER
+// XFAIL: *
+
+#include <atomic>
+#include <condition_variable>
+#include <future>
+#include <mutex>
+#include <thread>
+
+#include <CL/sycl.hpp>
+
+namespace S = cl::sycl;
+
+struct Context {
+  std::atomic_bool Flag;
+  S::queue &Queue;
```

Comment:
SYCL queue has reference semantics, taking a copy of it would be simpler.

Reply:
This test is removed.